teps/0003-tekton-catalog-organization.md

```diff
@@ -408,7 +408,60 @@ For example (with Task and Pipelines):
       /samples/dummy-go-release.yaml |
 ```
 
-An example of a *freezed* catalog migrated to this layout is available
+
+#### Example of a resource from catalog
+
+* An example of `git-clone` task from the catalog: [Git-Clone](https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1)
+
+* Fields added to the yaml file
+
+```yaml
+  labels:
+    app.kubernetes.io/version: "0.1"           👈 MUST: version of the resource
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"  👈 MUST: version of pipeline
+    tekton.dev/tags: git                       👈 Optional: Comma separated list of tags
+    tekton.dev/displayName: "git clone"        👈 Optional: Display name of the task
+spec:
+  description: >-
+    These Tasks are Git tasks to work with repositories used by other tasks
+    in your Pipeline.         👈 # MUST:  One line Summary of the task
+
+    The git-clone Task will clone a repo from the provided url into the
+    output Workspace. By default the repo will be cloned into the root of
+    your Workspace. You can clone into a subdirectory by setting this Task's
+    subdirectory param.       👈 # Optional: Descriiption
+```
+
+
+* Structure of the task:
+
+  ```
+  $ tree git-clone
+
+  git-clone       👈 # Directory of task name matches the resource name present in yaml file
+  └── 0.1
+      ├── git-clone.yaml     👈 # Filename of task name matches the resource name present in yaml file
+      ├── README.md
+      ├── samples
+      │   ├── git-cli
+      │   │   ├── pipeline.yaml
+      │   │   ├── pvc.yaml
+      │   │   ├── secret.yaml
+      │   │   └── service-account.yaml
+      │   ├── git-clone-checking-out-a-branch.yaml
+      │   ├── git-clone-checking-out-a-commit.yaml
+      │   ├── git-rebase
+      │   │   ├── run.yaml
+      │   │   ├── secret.yaml
+      │   │   └── service-account.yaml
+      │   └── using-git-clone-result.yaml
+      └── tests
+          └── run.yaml
+
+  ```
+
+An example of a *froze* catalog migrated to this layout is available
 [here](https://github.com/Pipelines-Marketplace/catalog).
```

Comment:
this does not reflect the exact changes for YAML now as `label` for a tag is changed from `tags` to `tekton.dev/tags`. Do we want to keep that in here? @sthaha

Reply:
@SM43 good point. lets remove this 🙏

Reply:
:+1: